pkg/env/base_image.go

```diff
@@ -13,4 +13,9 @@ var (
 	// BaseImageWatcherMaxConcurrentRepositories controls the maximum number of repositories
 	// processed concurrently during a poll cycle.
 	BaseImageWatcherMaxConcurrentRepositories = RegisterIntegerSetting("ROX_BASE_IMAGE_WATCHER_MAX_CONCURRENT_REPOSITORIES", 10)
+
+	// BaseImageWatcherRegistryRateLimit controls the maximum requests per second to each
+	// registry integration. The default of 5 req/s balances performance with rate limit safety.
+	// Lower to 1-2 for unauthenticated Docker Hub or aggressive rate-limited registries.
+	BaseImageWatcherRegistryRateLimit = RegisterIntegerSetting("ROX_BASE_IMAGE_WATCHER_REGISTRY_RATE_LIMIT", 5)
```

Comment:
nit: A config map entry could be a better place for this config?

Reply:
Likely. But I don't want to go into that route right now without thinking on the right experience for a config map here, which should include Sensor for delegated repository scanning/discovery.

Reply:
I will resolve this.